ts/WoltLabSuite/Core/Component/File/File.ts

```diff
@@ -0,0 +1,90 @@
+import WoltlabCoreFileElement from "WoltLabSuite/Core/Component/File/woltlab-core-file";
```

Comment:
The file name is too generic and it implies that it represents a fiel when it really just exposes a bunch of helper methods. It should be renamed to `Component/File/Helper.ts`.

Please remember to clean up the then obsolete JS artifact.